tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java

```diff
@@ -76,6 +76,9 @@
 import com.ibm.icu.util.VersionInfo;
 
 public class ShowLocaleCoverage {
+
+    private static final String TSV_BASE = "https://github.com/unicode-org/cldr-staging/blob/main/docs/charts/43/tsv/";
+
```

Comment:
For the future, can we get the "43" from a constant somewhere else (or create such if it does not exist)?

Reply:
Yes, I'll include that in another chart ticket.